src/scripts/main.js

```diff
@@ -1,3 +1,88 @@
 'use strict';
 
 // write code here
+const header = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const th = header.getElementsByTagName('th');
+const tr = tbody.getElementsByTagName('tr');
+const headers = [];
+const data = [];
+let sort = 'name|desc';
```

Comment:
looks like redundant.
the task is to implement asc order by click
![Screenshot from 2023-04-16 15-40-47](https://user-images.githubusercontent.com/33029864/232312266-687629aa-2617-4600-a1ce-e5d97274a05c.png)
now first click sort in desc order, second click sort in asc order,
it is reasont that all tests are failed